test/gtest/ucp/test_ucp_worker.cc

```diff
@@ -278,22 +309,28 @@ class test_ucp_worker_discard : public ucp_test {
     static       unsigned m_destroyed_ep_count;
     static       unsigned m_flush_ep_count;
     static       unsigned m_pending_add_ep_count;
+    static       ucp_ep_t m_fake_ep;
     static const unsigned m_pending_purge_reqs_count;
 
-    static std::vector<uct_completion_t*> m_flush_comps;
+    static std::vector<uct_completion_t*>  m_flush_comps;
     static std::vector<uct_pending_req_t*> m_pending_reqs;
-    static std::map<uct_ep_h, unsigned> m_pending_reqs_count;
+    static std::map<uct_ep_h,
+                    std::vector<uct_pending_req_t*> >
+                                           m_pending_reqs_map;
 };
 
 unsigned test_ucp_worker_discard::m_created_ep_count               = 0;
 unsigned test_ucp_worker_discard::m_destroyed_ep_count             = 0;
 unsigned test_ucp_worker_discard::m_flush_ep_count                 = 0;
 unsigned test_ucp_worker_discard::m_pending_add_ep_count           = 0;
+ucp_ep_t test_ucp_worker_discard::m_fake_ep                        = {};
 const unsigned test_ucp_worker_discard::m_pending_purge_reqs_count = 10;
 
-std::vector<uct_completion_t*> test_ucp_worker_discard::m_flush_comps;
+std::vector<uct_completion_t*>  test_ucp_worker_discard::m_flush_comps;
 std::vector<uct_pending_req_t*> test_ucp_worker_discard::m_pending_reqs;
-std::map<uct_ep_h, unsigned> test_ucp_worker_discard::m_pending_reqs_count;
+std::map<uct_ep_h,
+         std::vector<uct_pending_req_t*> >
```

Comment:
minor: can define type by `typedef` to make it shorter

Reply:
done